style.css

```diff
@@ -78,10 +77,10 @@ li {
   margin-bottom: 3px;
 }
 
-/* headline section styling */
+/* headline styles */
 .headline {
   overflow: hidden;
-  background-image: url(images/Group.png);
+  background-image: url(images/group.png);
   background-position-y: 125%;
   background-color: #1c1a19;
   display: flex;
```

Comment:
- [ ] It is noticed that `display: flex` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.flex-row` or `.flex-column` then assigning it wherever possible.
```
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```